### PR TITLE
Add Rust for C++ Programmers to Unofficial list

### PR DESCRIPTION
**Pull request by CPerezz**
The book does not pretend to be an introduction to Rust or to teach how to use it from scratch. But it contains really good resources on the memory management side. Extensive docs about `Rc`, `Arc, Allocation arenas and borrowing and other related topics.

### CODE REVIEW

File: src/unofficial.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Introductory:
 * [Easy Rust](https://dhghomon.github.io/easy_rust/) - aimed at non-English native speakers
 * [Learning Rust With Entirely Too Many Linked Lists](https://rust-unofficial.github.io/too-many-lists/)
 * [Rust Practise Questions](https://sn99.github.io/rust-practise-questions/) (sic)
+* [Rust for C++ Programmers](https://aminb.gitbooks.io/rust-for-c/content/borrowed/index.html)
```
Comment (glyn) [inline src/unofficial.md:11]: The link is part-way through the book:
```suggestion
* [Rust for C++ Programmers](https://aminb.gitbooks.io/rust-for-c/content/) - including a helpful section on [borrowed pointers](https://aminb.gitbooks.io/rust-for-c/content/borrowed/index.html)
```

**Review by glyn — CHANGES_REQUESTED** [review]
LGTM thanks, but I've made a suggestion so that this entry is consistent with the others, by linking to the start of the book.

**Comment by CPerezz** [comment]
OMFG.. Sorry I copied the incorrect link, the intention was to do it so that it was linked to the start page of the book. 
Should be fixed now. Apologies. @glyn  and appreciate your great work maintaining this source of knowledge!

**Review by glyn — APPROVED** [review]
LGTM. Thanks for the change.

**Pull request merged**